.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,38 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: 
+#     https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Upload Python Package
+
+on:
+  release:
+    types: [published]
+    branches: [main]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.9'
+    - name: Upgrade tooling
+      run: |
+        python3 -m pip install --upgrade pip
+        pip3 install --upgrade build setuptools setuptools_scm[toml] twine wheel
+    - name: Build package
+      run: |
+        python3 -m build --wheel
+        python3 setup.py sdist --format=zip
+    - name: Publish package
+      env:
+        TWINE_USERNAME: __token__
+        TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
Nice! Good we’re templating this in here.

Reply:
Thanks.